NikoBracket/CreateBracketViewController.m

```diff
@@ -175,24 +411,136 @@ - (UIView *)tableView:(UITableView *)tableView viewForHeaderInSection:(NSInteger
         header.textLabel.text = @"Midwest";
     }
     
+    if (self.roundControl.selectedSegmentIndex > 3) {
+        header.textLabel.text = @"";
+    }
+    
     header.textLabel.font = [header.textLabel.font fontWithSize:30];
     
     return header;
 }
 
+- (NSMutableArray *) initializeUserBracket:(NSMutableArray *) matchups {
+    NSMutableArray *round1 = matchups[0];
+    NSMutableArray *round2 = [[NSMutableArray alloc] init];
+    NSMutableArray *round3 = [[NSMutableArray alloc] init];
+    NSMutableArray *round4 = [[NSMutableArray alloc] init];
     
+    for (int i = 0; i < round1.count - 1; i+=2) {
+        NSArray *game = round1[i];
+        NSArray *game2 = round1[i+1];
 
+        NSString *winner1 = game[[game[2] intValue]];
+        NSString *winner2 = game2[[game2[2] intValue]];
```

Comment:
it would also be helpful to make clear what each `game` looks like, by adding a comment and explaining what each value in the array means